GameFiles/_integration.gsc

```diff
@@ -38,13 +34,35 @@ init()
 
     level.clientCommandCallbacks = [];
     level.clientCommandRusAsTarget = [];
+    level.overrideMethods = [];
+
+    wait 0.1;
+    InitializeGameMethods();
+
+    [[level.overrideMethods["SetDvarIfUninitialized"]]]( level.eventBus.inVar, "" );
+    [[level.overrideMethods["SetDvarIfUninitialized"]]]( level.eventBus.outVar, "" );
+    [[level.overrideMethods["SetDvarIfUninitialized"]]]( "sv_iw4madmin_integration_enabled", 1 );
+    [[level.overrideMethods["SetDvarIfUninitialized"]]]( "sv_iw4madmin_integration_debug", 0 );
+
+    level.logger = spawnstruct();
+    InitializeLogger();
+
+    //The gamename dvar seems to be an IW only thing..
+    if (level.eventBus.gamename == "") {
+        version = getDvar( "version" );
```

Comment:
There is a better way to do this.
`protocol` is an integer dvar and does not require you to make string comparison and is definitely better than this.
I have attached an image containing the default value on
![Capture](https://user-images.githubusercontent.com/37080671/180640219-44e2b451-bdf7-46da-a19c-035503a1a19c.PNG)
 T5. Check that plutonium is not overriding it.

Reply:
After discussing this with @RaidMax each game will get its own gsc file and will set the variable from there.